crates/stark-sdk/Cargo.toml

```diff
@@ -36,6 +36,16 @@ tracing-subscriber = { version = "0.3.17", features = ["std", "env-filter"] }
 tracing-forest = { version = "0.1.6", features = ["ansi", "smallvec"] }
 metrics-tracing-context = "0.16.0"
 metrics-util = "0.17.0"
+metrics-exporter-prometheus = { version = "0.15.3", features = [
```

Comment:
hm let's make these (including tokio below etc) optional and gate under some "prometheus" feature